lib/adyen/utils/hmac_validator.rb

```diff
@@ -0,0 +1,48 @@
+module Adyen
+  module Utils
+    class HmacValidator
+      HMAC_ALGORITHM = 'sha256'.freeze
+      DATA_SEPARATOR = ':'.freeze
+      VALIDATION_KEYS = %w[
+        pspReference originalReference merchantAccountCode merchantReference
+        amount.value amount.currency eventCode success
+      ].freeze
+
+      def validate(notification_request_item, hmac_key)
```

Comment:
Could you rename this to is_valid_notification_hmac ?

Reply:
If you don't mind, I would like to suggest naming it to `valid_notification_hmac?` instead, this is the preferred way to define methods that return booleans.

ref: https://rubystyle.guide/#bool-methods-qmark